src/wp-includes/ms-settings.php

```diff
@@ -54,15 +54,23 @@
 // have not been populated in the global scope through something like `sunrise.php`.
 if ( ! isset( $current_site ) || ! isset( $current_blog ) ) {
 
-	$domain = strtolower( stripslashes( $_SERVER['HTTP_HOST'] ) );
-	if ( ':80' === substr( $domain, -3 ) ) {
-		$domain               = substr( $domain, 0, -3 );
-		$_SERVER['HTTP_HOST'] = substr( $_SERVER['HTTP_HOST'], 0, -3 );
-	} elseif ( ':443' === substr( $domain, -4 ) ) {
-		$domain               = substr( $domain, 0, -4 );
-		$_SERVER['HTTP_HOST'] = substr( $_SERVER['HTTP_HOST'], 0, -4 );
+	/**
+	 * Filters allowed HTTP ports in Multisite.
+	 *
+	 * @param array $allowed_ports The allowed ports. Default is [ ':80', ':443' ].
+	 *
+	 * @since 5.3.0
+	 *
```

Comment:
```suggestion
	 * @since 5.3.0
	 *
	 * @param array $allowed_ports The allowed ports. Default is [ ':80', ':443' ].
```

- `@param` then `@since`.
- Removed extra DocBlock line.

Note: If the ticket is milestoned, this `@since` annotation will need to be updated to match the Milestone.